### DISCUSSION TITLE
New uno Project xaml files telling about inconsistent line endings?

### DISCUSSION THREAD

**Discussion by DevTKSS** [unoplatform/uno#18989] [Q&A]
dont know if its a uno only thing but my visual studio 2022 telling me every new project opening a xaml file in it:
![image](https://github.com/user-attachments/assets/93c97d27-352f-444a-940c-71e7b58af509)
My vs UI is set to german, but raw translated it is telling me everytime about incosistent line endings which I have to check for each file and looking for a way it might do this on its own on creation of new things of this. If ticking "dont show again" i am afraid it will just be left without knowing this answer if needed.
Can someone tell me the solution for this or maybe my vs setup is lacking of this setting which makes it ask me each time.
Issue for me on uno project is, the project is building, but opening the xaml file it is dont recognizing some things. answering this dialog form and restarting vs it is most of the time not longer having that issue, but a solution to not have to answer it for every xaml on new ones would be great so not having to do this workaround.

for example of the not recognized things this is causing:
![Screenshot 2024-12-03 114228](https://github.com/user-attachments/assets/5a2da822-e019-47be-866a-754ae0fd9c94)
![Screenshot 2024-12-03 114244](https://github.com/user-attachments/assets/0baa11a1-695c-4ccf-b106-0d9804bf0915)
![Screenshot 2024-12-03 114303](https://github.com/user-attachments/assets/5801e42d-8f54-4001-b311-25684610663d)

**Comment by jeromelaban** [comment]
Thanks for the report!

Indeed, there may be line ending changes that are incorrect in the templates. 

How did you create this specific file? Is it using the add new item UI in VS2022?

**Reply by DevTKSS** [reply]
Actually that is happening when I create a new project with the uno wizard inside of Visual Studio 2022 Community Edition...

**Reply by MartinZikmund** [reply]
Created https://github.com/unoplatform/uno.templates/issues/1157 to track this
